lib/retrieve_title.rb

```diff
@@ -3,15 +3,15 @@
 module RetrieveTitle
   CRAWL_TIMEOUT = 1
 
-  def self.crawl(url, max_redirects: nil, initial_https_redirect_ignore_limit: false)
+  def self.crawl(url, max_redirects: nil, initial_https_redirect_ignore_limit: false, log_errors: true)
     fetch_title(
       url,
       max_redirects: max_redirects,
       initial_https_redirect_ignore_limit: initial_https_redirect_ignore_limit
     )
   rescue Exception => ex
+    Rails.logger.error(ex) if log_errors
     raise if Rails.env.test?
```

Comment:
no need to change this now, but it does feel like a mighty red flag to me. Fail fast in test, but in production ... eat it. makes our specs lie.

Reply:
actually ... now the new specs have non production behavior, I think we are going to have to nuke this line. Maybe instead

( ... log_errors: true, raise_errors: false) 

then fix usage in specs.